help/destinations/catalog/streaming/tiktok.md

```diff
@@ -0,0 +1,114 @@
+---
```

Comment:
Two general comments: 

1. I notice that you placed the TikTok documentation page within our streaming section. Are you sure that 
For reference, other social media destinations in our catalog are located either within the Advertising or the Social categories (see screenshot below). Unfortunately, the UI does not currently support multiple tagging, so you will need to choose one category only. 

![Screen Shot 2023-02-01 at 3 28 09 PM](https://user-images.githubusercontent.com/19704189/216156132-48ad6eaf-7d97-494c-bbe6-b5bae009cc36.png)

2. I noticed that your created a folder for your screenshots in destination-sdk/docs-framework. The images should be placed in a separate assets folder. Please see instructions [here](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/use-github-interface-to-create-documentation.html?lang=en#author-documentation) if you are using the GitHub web interface (step 3) or [here](https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/work-in-local-environment.html?lang=en) if you are using a text editor (step 4).